src/components/Episode/Episode.jsx

```diff
@@ -21,8 +31,17 @@ export default function Episode(props){
 
 	const { videoSrc } = video;
 
+	const isActive = index === activeIndex;
+
 	return (
-		<li>
+		<li
+			id={ id }
+			className={ s.wrapper }
+			role="group"
+			aria-roledescription="slide"
+			aria-label={ `Episode ${number} of ${data.length}.` }
```

Comment:
in hindsight this feels kinda weird to access the data directly - maybe we could include the number of episodes in the provider?

Reply:
Changed so we have an episodeCount property in our context provider. 
Worth noting that we could do the same thing for Characters, i chose not to as this branch is episodes only. Let me know if it's fine practice to make that change and I can.

Reply:
feel free to make the changes on `markup` and just use the same implementation - maybe branch off, but no need to do a PR if the implementation is the same 👌